.github/workflows/docs.yml

```
@@ -0,0 +1,125 @@
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+name: Docs
+
+jobs:
+  set-image:
```

```suggestion
  set-image:
    # GitHub Actions allows using 'env' in a container context.
    # However, env variables don't work for forks: https://github.com/orgs/community/discussions/44322
    # This workaround sets the container image for each job using 'set-image' job output.  
    # TODO: remove once migration is complete or this workflow is fully stable
    if: contains(github.event.label.name, 'GHA-migration')
```
Adds comment why `set-image` is needed and also condition because we are not moving it yet to GHA